### DISCUSSION TITLE
Add per habit option to check previous day from notification

### DISCUSSION THREAD

**Discussion by licaon-kter** [iSoron/uhabits#220] [Feature Requests]
I have notifications in the morning for the "previous day", but currently I can't actually use the notification actions as these pertain to "current day" only.

Not sure if #50 covers this too.

**Comment by iSoron** [comment]
Even if you wait until the next day to click the notification, the checkmark will still appear on the date the notification first showed up. For example, if the notification appears on Saturday, but you only click it on Sunday, a checkmark will be added on Saturday.

**Comment by licaon-kter** [comment]
Umm no, it's notification "**for** previous day", not "**from** previous day".
Notification today that checks for yesterday.
